chatwoot/chatwoot
# feat: Add campaigns in web widget

- Fetch campaigns 
- Store campaigns
- Timer for executing campaigns
- API call to events endpoint

## How to Test
1. Enable inbox campaigns

Open `app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue`
Add the code on line number 337

```
{
 key: 'campaign',
 name: this.$t('INBOX_MGMT.TABS.CAMPAIGN'),
}
```
2. Go to any website inbox

3. Click on the settings

4. Go to the campaigns tab

5. Create a new campaign by entering details title, messages, sender, URL and time on page

6. Open the webpage which configured website inbox

7. Wait for the time specified in the campaign

8. Check messages are coming over the widget
